openedx/features/teams/decorators.py

```diff
@@ -1,12 +1,16 @@
+""" Decorators for teams application """
 from django.core.urlresolvers import reverse
 from django.shortcuts import redirect
-from lms.djangoapps.courseware.courses import get_course_by_id
 from opaque_keys.edx.locations import SlashSeparatedCourseKey
 
+from lms.djangoapps.courseware.courses import get_course_by_id
+
 
 def can_view_teams(function):
-    def wrap(request, *args, **kwargs):
-
+    """ If course has ended redirect user to teams dashboard, otherwise proceed normally. For the parameters,
+    see PEP 318
+    """
+    def wrap(request, *args, **kwargs):  # pylint: disable=missing-docstring
```

Comment:
Why did we have to add this directive here? `pylint: disable=missing-docstring`

Reply:
We are suppression linitng warning to add docstring for `wrap` function. `wrap` is not our function, we must override it to create a new decorator. So, there is no point to add its docstring here and also for its params `request, *args, **kwargs` 

This intent of decorator has already been documented in `can_view_teams`